cmd/snap-seccomp-blacklist/Makefile

```diff
@@ -0,0 +1,40 @@
+.PHONY: all
+all: $(foreach v,LE BE,$v-blacklist.bpf) | analyze
+
+.PHONY: clean
+clean:
+	rm -f snap-seccomp-blacklist snap-seccomp-blacklist.o *.pfc *.bpf
+
+.PHONY: fmt
+fmt: snap-seccomp-blacklist.c
+	clang-format -style='{BasedOnStyle: Google, IndentWidth: 4, ColumnLimit: 120}' -i $^
+
+DESTDIR ?=
```

Comment:
Mostly curious, this looks a bit unusual; if it's not set why set it to empty, will make complain if unset?

Reply:
Yes, make does complain if you ask it to via `--warn-undefined-variables`. I do it regularly to avoid silly mistakes.